changelogs/fragments/deprecate-ansible-2.9-2.10.yml

```
@@ -0,0 +1,6 @@
+deprecated_features:
+  - "Support for Ansible 2.9 and ansible-base 2.10 is deprecated, and will be removed in the next major release (community.general 5.0.0) in next spring.
```

```suggestion
  - "Support for Ansible 2.9 and ansible-base 2.10 is deprecated, and will be removed in the next major release (community.general 5.0.0) next spring.
```
extremely tiny wording nit